spylib/token.py

```diff
@@ -216,14 +238,42 @@ async def execute_gql(
         return jsondata['data']
 
 
-class UniqueStore(Store):
-    __instance = None
+class OfflineTokenABC(Token, ABC):
+    """
+    Offline tokens are used for long term access, and do not have a set expiry.
+    """
+
+    @abstractmethod
+    async def save_token(self):
+        pass
+
+    @abstractclassmethod
+    def load_token(cls, store_name: str):
```

Comment:
Can we just call them `save` and `load` since they will be called on a token instance?
I'm lazy that way

Reply:
This method has less comments than the OnlineToken equivalent ... it's a little jealous ...  :wink: